docs/docs/segments/time.md

```diff
@@ -21,6 +21,19 @@ Show the current timestamp.
 }
 ```
 
+### Display time in multiple time zones with [Sprig](https://masterminds.github.io/sprig/date.html)
```

Comment:
- Would it be an idea to add an `## Examples` section below Properties, where this is one of the examples? I feel like we want to have people read through to the bottom and then see how to use certain functionality.
- Links always need to be at the bottom, see other link examples in this file.

Reply:
The PR definitely was meant to spark a discussion, I appreciate the feedback. `Examples` is a great section to add across the documentation- Git's documentation [has it](https://git-scm.com/docs/git-restore#_examples) and a good sense of familiarity always enhances usability. Note, the headings there are also linked via named anchors -- makes sections easier to crosss-reference and share - I like that feature as well. What do you think?